charts/metrics-server/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@
 - Fixed nanny's RoleBinding which contained a hard-coded namespace instead of the Helm's release namespace. ([#1479](https://github.com/kubernetes-sigs/metrics-server/pull/1479)) _@
 the-technat_
 
+## [3.12.2] - TBC
+
+### Changed
```

Comment:
```suggestion
### Changed
```
The changes should be added under the `[UNRELEASED]` heading as we don't know which chart version the changes will be released in until a release is requested.

Reply:
Fixed